README.md

```diff
@@ -4,11 +4,16 @@ This project involves working with the real live data from the SpaceX API. The t
 
 ## Project Snapshot
 
+<img width="1440" alt="Screenshot 2022-09-30 at 1 54 10 PM" src="https://user-images.githubusercontent.com/48407831/193274365-c72418fb-06e0-4e7f-a068-36da4de4450c.png">
 
+<img width="1438" alt="Screenshot 2022-09-30 at 1 54 47 PM" src="https://user-images.githubusercontent.com/48407831/193274612-6d6a2d09-a30f-4f66-b802-82de09cf6b3d.png">
+
+<img width="1440" alt="Screenshot 2022-09-30 at 1 55 10 PM" src="https://user-images.githubusercontent.com/48407831/193274688-2013df86-2c5f-4808-b166-cfbb18a7ac82.png">
```

Comment:
Since this is a group project kindly make sure that the information under the `authors` section on the README includes all collaborators.

Reply:
Fixed